src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java

```diff
@@ -331,22 +331,30 @@ private void addSkills(Hero hero, HeroClass heroClass, Collection<String> skillS
         if (hero == null) return emptySkillList;
 
         HeroClass heroClass = hero.getHeroClass();
-        HeroClass secondClass = hero.getSecondaryClass();
+        HeroClass secondClass = hero.getSecondaryClass();();
```

Comment:
This looks weird, does it compile? Is (); a valid statement in Java?

Reply:
Oh geez... I dont know what happened with this, I guess I copy/pasted wrong. Fixing.